src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/Reference.vue

```diff
@@ -1,7 +1,13 @@
 <template>
   <Wizard
     :ref="'wizard'"
-    :wizard="wizardStore.wizardData?.inviteType === PortalInviteType.WORK_EXPERIENCE ? workExperienceReferenceWizardConfig : characterReferenceWizardConfig"
+    :wizard="
+      wizardStore.wizardData?.inviteType === PortalInviteType.WORK_EXPERIENCE
+        ? wizardStore.wizardData.workExperienceType == WorkExperienceType.IS_400_Hours
+          ? workExperienceReference400HoursWizardConfig
+          : workExperienceReferenceWizardConfig
+        : characterReferenceWizardConfig
+    "
```

Comment:
I have a good solution to simplify this. 

We can copy what we did in Application.vue 

Where we basically take all this config setting logic, and put it into line 98 where we determine which intitializeWizard to use. That way you don't have to write the logic twice. 

(I did it this way initially since we only had Character + Work Experience). Simpler times back then.

Reply:
good catch! Thanks!